seqr/views/apis/project_api_tests.py

```diff
@@ -96,6 +97,9 @@ def test_project_page_data(self):
             response_json['projectsByGuid'][PROJECT_GUID]['lastAccessedDate'][:10],
             datetime.today().strftime('%Y-%m-%d')
         )
+        collaborators = [{k: c[k] for k in c if k != 'lastLogin'}  # remove key 'lastLogin' for the time keeps changing
```

Comment:
if in the `PROJECT_COLLABORATORS` constant you define `'lastLogin': mock.ANY` it will work and be more readable here

Reply:
Good point.